internal/release/version-build

```diff
@@ -1 +1 @@
-0010200
\ No newline at end of file
+0010301
```

Comment:
should this be 0010301 or 0010300 - I see in the diff that we had build version 00 before.

Reply:
Build numbers should start at 1 - the BN before didn't abide by this rule. That being said, server wouldn't reject either, but would be nice to stay consistent with how other 1P clients do it.